CONTRIBUTING.md

```diff
@@ -39,23 +39,47 @@ contribute is to learn how to use it and put your changes on a Git repository.
 There's a plenty of documentation about Git -- you can start with the [Pro Git
 book](http://git-scm.com/book/).
 
-### Forks + GitHub Pull requests
+
+### Setting up a Development Environment
+
+To set up your local development environment for contributing to the main
+repository [nltk/nltk](https://github.com/nltk/nltk/):
+
+- Fork the [nltk/nltk](https://github.com/nltk/nltk/) repository on GitHub
+  to your account;
+- Clone your forked repository locally
+  (`git clone https://github.com/<your-github-username>/nltk.git`);
+- Run `cd nltk` to get to the root directory of the `nltk` code base;
+- Install the dependencies (`pip install -r pip-req.txt`);
+- Install the `nltk` library in development mode
+  (`python setup.py develop`);
```

Comment:
I don't think there's a need to install the `nltk` in development mode but I might be wrong. I've been testing with `tox` without installing and letting it override my environment's `nltk`.

Reply:
tl;dr -- I'm removing this `python setup.py develop` line. Simpler is better. :-)

I'm not entirely sure either, but I have the opposite situation (and see it with co-workers as well): We install it in the dev mode so that we can use the library not necessarily from the repo root directory. If we want the upstream official release version of the library, we check out the `develop` branch. If we want to use or test whatever new feature being developed, we check out the relevant branch instead. Now that I've spelled this out, I see the advantage of this workflow is to avoid potential confusion by having only one installation of `nltk` (this is the part that's opposite of what you achieve for not overriding the installed, official version by not installing a dev version), with the trade-off of slightly more git work for always committing work in progress to allow the back-and-forth branch switching. That said, I think it'd be better to keep things simple for the purpose of getting more people started with contributing to NLTK. And I think you're right that `tox` should work without installing `nltk`, if one is at the repo root directory where the directory `nltk/` is available as the local relative-path package.